nncf/experimental/post_training/algorithms/algorithm.py

```diff
@@ -69,8 +84,25 @@ def get_statistic_points(self, model: ModelType) -> StatisticPointsContainer:
         Returns activation layers, for which StatisticsCollector should collect statistics.
         """
 
-    @abstractmethod
+
+class ComplexAlgorithm(Algorithm):
```

Comment:
Should be `CompositeAlgorithm` since we use the word "composite" in the names of other classes in NNCF with the same purpose.

Reply:
Done